kubechain/hack/humanlayer_client_play.go

```diff
@@ -0,0 +1,66 @@
+package main
```

Comment:
i like this pattern...should probably get its own folder so other go scripts can be dropped in here without colliding `main()`

Reply:
(not a blocker to merge)